sys/net/gnrc/Makefile.dep

```diff
@@ -246,8 +246,8 @@ endif
 ifneq (,$(filter gnrc_sixlowpan_iphc,$(USEMODULE)))
   USEMODULE += gnrc_ipv6
   USEMODULE += gnrc_sixlowpan
-  USEMODULE += gnrc_sixlowpan_ctx
   USEMODULE += gnrc_sixlowpan_iphc_nhc
+  DEFAULT_MODULE += gnrc_sixlowpan_ctx
```

Comment:
This module has dependencies though, I'm not sure it should be a default module, it will make the build-depend on inclusion order.

Reply:
Also `DEFAULT_MODULE` are only evaluated before the dependency recursions and after it ends, this won't be activated in most cases because of that

Reply:
hm so'd have to add a pseudo-module `gnrc_sixlowpan_no_ctx` instread?

Reply:
I was going to say some about having to include a negative module instead of just modelling the `gnrc_sixlowpan_ctx` but I guess this just a workaround for build system limitations?

This should be noted for when kconfig modelling gets around to this.